SWLOR.Game.Server/Feature/SpawnDefinition/DantooineSpawnDefinition.cs

```diff
@@ -0,0 +1,201 @@
+                .RandomlyWalks()
+                .WithFrequency(100)
+            .ReturnsHome();
+        }
+
+        private void Janta()
+        {
+            _builder.Create("DANTOOINE_JANTA")
+                .AddSpawn(ObjectType.Creature, "gizka")
+                .RandomlyWalks()
+                .WithFrequency(10)
+
+                .AddSpawn(ObjectType.Creature, "voritorlizard")
+                .RandomlyWalks()
+                .WithFrequency(50)
+            .ReturnsHome();
+        }
+
+        private void DanBoss()
+        {
+            _builder.Create("DANTOOINE_JANTA_BOSS")
+                .AddSpawn(ObjectType.Creature, "vangrykath")
+                .RandomlyWalks()
+                .WithFrequency(1)
+                .RespawnDelay(60 + Random.D100(1))
+            .ReturnsHome();
+        }
+
+        private void DanBoss2()
+        {
+            _builder.Create("DANTOOINE_KIN_BOSS")
+                .AddSpawn(ObjectType.Creature, "vqueenkin")
+                .RandomlyWalks()
+                .WithFrequency(1)
+                .RespawnDelay(60 + Random.D100(1))
+            .ReturnsHome();
+        }
+
+        private void DantooineMountains()
+        {
+            _builder.Create("DANTOOINE_MOUNTAINS")
+                .AddSpawn(ObjectType.Creature, "Iriaz")
+                .RandomlyWalks()
+                .WithFrequency(100)
+                .ReturnsHome()
+
+                .AddSpawn(ObjectType.Creature, "Thune")
+                .RandomlyWalks()
+                .WithFrequency(100)
+            .ReturnsHome();
+        }
+
+        private void DantooineCanyon()
+        {
+            _builder.Create("DANTOOINE_CANYON")
+                .AddSpawn(ObjectType.Creature, "gizka")
+                .RandomlyWalks()
+                .WithFrequency(80)
+                .ReturnsHome()
+
+                .AddSpawn(ObjectType.Creature, "Iriaz")
+                .RandomlyWalks()
+                .WithFrequency(50)
+                .ReturnsHome();
+        }
+
+        private void DantooineSouthPlains()
+        {
+            _builder.Create("DANTOOINE_SOUTH_PLAINS")
+                .AddSpawn(ObjectType.Creature, "dantarihunter")
+                .RandomlyWalks()
+                .WithFrequency(100)
+
+                .AddSpawn(ObjectType.Creature, "jantashaman")
+                .RandomlyWalks()
+                .WithFrequency(100)
+            .ReturnsHome();
+        }
+
+        private void DantooineCrystalCaves()
+        {
+            _builder.Create("DANTOOINE_CRYSTAL_CAVES")
+                .AddSpawn(ObjectType.Creature, "voritorlizardbos")
+                .RandomlyWalks()
+                .WithFrequency(100)
+
+                .AddSpawn(ObjectType.Creature, "gizka")
+                .RandomlyWalks()
+                .WithFrequency(100)
+            .ReturnsHome();
+        }
+
+        private void DantooineNpcs()
+        {
+            _builder.Create("DANTOOINE_NPCS")
```

Comment:
If you want random farmers spawned, that's fine. But an alternative is to simply place them in the area. They won't be random but it's a little easier than creating a spawn table for them. This is acceptable, just letting you know of other options.